doc/man3/X509_STORE_CTX_new.pod

```diff
@@ -153,13 +152,13 @@ should be made or reference counts increased instead.
 
 =head1 RETURN VALUES
 
-X509_STORE_CTX_new() returns a newly allocates context or B<NULL> is an
+X509_STORE_CTX_new() returns a newly allocates context or NULL is an
```

Comment:
Heh, two typos to fix when you're touching this text - `allocated`, `if`

Reply:
Sure, done.